blockchain/chain/src/store/chain_store.rs

```diff
@@ -778,8 +778,8 @@ where
         .transpose()?)
 }
 
-/// Attempts to deserialize to unsigend message or signed message and then returns it at as a
-// [ChainMessage].
+/// Attempts to de-serialize to unsigned message or signed message and then returns it as a
```

Comment:
I'm so happy when typos like this are fixed. It's like we're real software now. :)

Reply:
Exactly, and good job for the setup of this tool!